connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1694,33 +1702,41 @@ ble_error_t Gap::createAdvertisingSet(
 #if BLE_FEATURE_EXTENDED_ADVERTISING
 ble_error_t Gap::destroyAdvertisingSet(advertising_handle_t handle)
 {
+    tr_info("destroying advertising set %d", handle);
     if (is_extended_advertising_available() == false) {
+        tr_error("extended advertising not available");
         return BLE_ERROR_OPERATION_NOT_PERMITTED;
     }
 
     if (handle == LEGACY_ADVERTISING_HANDLE) {
+        tr_error("handle cannot be equal to %d", LEGACY_ADVERTISING_HANDLE);
```

Comment:
```suggestion
        tr_error("Legacy advertising set cannot be destroyed");
```

Reply:
I like the change @paul-szczepanek-arm but it is valuable to print the handle value.